frontera/core/models.py

```diff
@@ -22,7 +22,7 @@ def __init__(self, url, method=b'GET', headers=None, cookies=None, meta=None, bo
         :param string method: HTTP method to use.
         :param dict headers: dictionary of headers to send.
         :param dict cookies: dictionary of cookies to attach to this request.
-        :param dict meta: dictionary that contains arbitrary metadata for this request.
+        :param dict meta: dictionary that contains arbitrary metadata for this request, both keys and values should be bytes.
```

Comment:
not exactly.
Values must be either bytes or serializable objects (e.g. lists, tuples, dicts) with all items of type bytes.